docs/topics/logging.rst

```diff
@@ -194,6 +195,18 @@ If :setting:`LOG_SHORT_NAMES` is set, then the logs will not display the Scrapy
 component that prints the log. It is unset by default, hence logs contain the
 Scrapy component responsible for that log output.
 
+If :setting:`LOG_TO_SYSTEMD` is set to ``True``, Scrapy will send logs to the
+systemd journal using ``systemd.journal.JournalHandler`` as its logging handler.
+This requires the external Python package ``systemd`` to be installed (minimum
+version 234 recommended). You can install it via pip::
+
+    pip install systemd
+
+This feature is opt-in and disabled by default. When enabled, Scrapy will raise an
+``ImportError`` if the ``systemd`` package is not found. Use this setting when
+running Scrapy under systemd to utilize ``journalctl`` for log management and
+filtering.
```

Comment:
You can have a minor reference here, but the setting documentation should live in the settings.rst page, with a section of its own properly labeled so that ``` :setting:`FOO` ``` actually links there. Consider running `tox -e docs` to check how your docs render.